client/src/ui/molecules/manage-upgrade-dialog/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useLocale } from "../../../hooks";
+import { Button } from "../../atoms/button";
+import { Icon } from "../../atoms/icon";
+import "./index.scss";
+
+export function ManageOrUpgradeDialogNotifications({ setShow }) {
+  const locale = useLocale();
+
+  return (
+    <div
+      className={`manage-upgrade-dialog show`}
```

Comment:
```suggestion
      className={`watch-submenu manage-upgrade-dialog show`}
```
This makes the buttons look aligned on mobile 🙈